packages/main/test/specs/Input.spec.js

```diff
@@ -162,6 +162,67 @@ describe("Input general interaction", () => {
 		assert.strictEqual(await inputChangeResult.getValue(), "2", "change is called twice");
 	});
 
+	it("Change event behaviour when focusing", async () => {
+		// Setup
+		const input = await browser.$("#myInput").shadow$("input");
+		const focusoutInput = await browser.$("#myInput2").shadow$("input");
+		const changeCount = await browser.$("#myInput-change-count");
+
+		// Act
+		await input.click();
+		await input.keys("z");
+		await focusoutInput.click();
+
+		// // Assert
+		assert.strictEqual(await changeCount.getHTML(false), "1", "The change event is called");
+
+		// Act
+		await input.click();
+		await focusoutInput.click();
+
+		// Assert
+		assert.strictEqual(await changeCount.getHTML(false), "1", "The change event is not called again");
+
+		// Act
+		await input.click();
+		await input.keys("f");
+		await focusoutInput.click();
+
+		// Assert
+		assert.strictEqual(await changeCount.getHTML(false), "2", "The change event is called for the changed value");
+	});
+
+	it("Change event behaviour when focusing + ENTER", async () => {
```

Comment:
Please make the two tests independent (reopen the page in the beginning of the test and adjust counter)

Reply:
Of course, we don't need to rely on the previous state, but the whole test file is made in that manner. I'll try to fix it.

Reply:
Done